src/convertToObject.js

```diff
@@ -11,8 +12,19 @@
  *
  * @return {object}
  */
+
 function convertToObject(sourceString) {
-  // write your code here
+  const sourceObj = sourceString.split(';')
+    .map(source => source.trim())
```

Comment:
Checklist - 4 [CODE STYLE] - All chained methods called on the same level should have same indentation. This makes code much more readable.
(move `.split(';')` to a new line)